sites/org/pages/new/pattern/[design]/[type]/[id].mjs

```diff
@@ -14,7 +13,7 @@ import { Null } from 'shared/components/null.mjs'
 import { DynamicOrgDocs as DynamicDocs } from 'site/components/dynamic-org-docs.mjs'
 
 // Translation namespaces used on this page
-const namespaces = [...new Set(['aaron', ...wbNs, ...pageNs])]
+const namespaces = [...new Set([...wbNs, ...pageNs])]
```

Comment:
Since this is a common pattern, I've added the `nsMerge()` method to `shared/utils.mjs` which does this. (merge all namespaces without dupes)

Reply:
This is FYI, not blocking the merge because of this